GG/src/Menu.cpp

```diff
@@ -465,7 +465,9 @@ void PopupMenu::Render()
             }
             std::vector<Font::LineData> lines;
             Flags<TextFormat> fmt = FORMAT_LEFT | FORMAT_TOP;
-            Pt menu_sz = m_font->DetermineLines(str, fmt, X0, lines); // get dimensions of text in menu
+            std::vector<boost::shared_ptr<Font::TextElement> > text_elements
+                = m_font->ComputeTextElements(str, fmt);
+            Pt menu_sz = m_font->DetermineLines(str, fmt, X0, text_elements, lines); // get dimensions of text in menu
```

Comment:
I have already added the following comment in the documentation of the definition of the `DetermineLines()` near [Font.h:486](https://github.com/LGM-Doyle/freeorion/blob/22632e325886231a2eb0a57cbdc58280d6361f20/GG/GG/Font.h#L486)

`A \p box_width of X0 will add a line break at every whitespace element in \p text_elements.`

I was not clear about that in the previous comment.

Do you also want an additional comment at each invocation of the function that uses `X0` as a value?